src/plugins/condition/Condition.js

```diff
@@ -72,7 +73,11 @@ export default class ConditionClass extends EventEmitter {
             return;
```

Comment:
It took me a while to figure out what this function is doing. I'm a little slow sometimes 😄  Some things that I think would help slow people me -

1. In my opinion, event handlers should be named so that they describe what they do (that it "handles" an event goes without saying - what else does it do?)
2. I think that the `broadcastTelemetry` event should be more descriptively named. Why are we broadcasting telemetry? I had to look at the code to confirm my hunch that this is called when new telemetry is received. Perhaps we could rename it `telemetryReceived`. 
3. `handleSubscription` sounds like we're handling some subscription lifecycle event. It wasn't clear to me that this function was for handling newly received telemetry.
4. Why, after receiving new telemetry do we emit a new event named `subscription`?

I'll file a followup issue to address this, and some of the other broader refactoring I think we need. No need to address it right now.

Reply:
@davetsay Let's do this refactor together?